single_stage_detector/ssd/utils.py

```diff
@@ -303,8 +301,6 @@ def __init__(self):
         # Implementation uses 1 iteration to find a possible candidate, this
         # was shown to produce the same mAP as using more iterations.
         self.num_cropping_iterations = 1
-        mlperf_log.ssd_print(key=mlperf_log.NUM_CROPPING_ITERATIONS,
```

Comment:
I believe this v0.5 key should not be deleted, but rather replaced by the `mllog_const.MAX_SAMPLES` key.  (The `mllog_const.MAX_SAMPLES` key mentioned at https://github.com/mlperf/training/pull/367/files#diff-591431110d6b55f5afe595b96253fddbR113)